title: [ch-set dotenv]

nili-alu opened the pull request:
use dotenv file to hide sensitive information
Delivers #37

nili-alu committed:
title: [ch-set dotenv]
use dotenv file to hide sensitive information
Delivers #37

chanceDira reviewed (changes requested):
Good progress, just some improvements
- How do we know if any of the environment variables are changed?
- How do I test this PR, and improve your PR description?
- Let's add a .env.example file that shows some environment variables only without their actual values.

nili-alu committed:
Update README.md

nili-alu committed:
Update README.md

vanessa-abewe committed:
Title: [setup dotenv]
tested whether the variables has changed
How should this be mannually tested
[by running node index.js]
delivery #37

ALVINdimpos commented on index.js:
```diff
@@ -0,0 +1,36 @@
+    console.log("The environment variable has not changed");
+}
+//
+if (process.env.DATABASE !== dbName) {
```
how we will use this if condition when it's merged?
